driver/ni/task.cpp

```diff
@@ -80,7 +79,8 @@ bool ni::ScannerTask::ok() {
 }
 
 ni::ScannerTask::~ScannerTask() {
-    if(this->thread->joinable() && (this->thread->get_id() != std::this_thread::get_id())) {
+    if (this->thread->joinable() && (
+            this->thread->get_id() != std::this_thread::get_id())) {
```

Comment:
the paranthesis on its own line is weird fix

Reply:
Fixed